ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+    "   3. TODO\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "\n",
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import eups.setupcmd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I: Interacting with data. Introduction to the Butler\n",
+    "https://pipelines.lsst.io/getting-started/data-setup.html\n",
+    "\n",
+    "Part I runs the following command-line tasks\n",
+    "```\n",
+    "eups list lsst_distrib\n",
+    "setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "echo \"lsst.obs.hsc.HscMapper\" > /home/jmyles/DATA/_mapper\n",
+    "ingestImages.py /home/jmyles/DATA /home/jmyles/repositories/ci_hsc/raw/*.fits --mode=link\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/CALIB/ /home/jmyles/DATA/CALIB\n",
+    "installTransmissionCurves.py /home/jmyles/DATA\n",
+    "mkdir -p /home/jmyles/DATA/ref_cats\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/ps1_pv3_3pi_20170110 /home/jmyles/DATA/ref_cats/ps1_pv3_3pi_20170110\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!eups list lsst_distrib\n",
+    "datarepo = \"/home/jmyles/repositories/ci_hsc/\"\n",
+    "datadir = \"/home/jmyles/DATA/\"\n",
+    "os.system(\"mkdir -p {}\".format(datadir))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "\n",
+    "setup = eups.setupcmd.EupsSetup([\"-j\",\"-r\", datarepo])\n",
+    "status = setup.run()\n",
+    "print('setup exited with status {}'.format(status))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Butler needs a *mapper* file \"to find and organize data in a format specific to each camera.\" We write this file to the data repository so that any instantiated Butler object knows which mapper to use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(datadir + \"_mapper\", \"w\") as f:\n",
+    "    f.write(\"lsst.obs.hsc.HscMapper\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ingest script\n",
+    "!ingestImages.py /home/jmyles/DATA /home/jmyles/repositories/ci_hsc/raw/*.fits --mode=link"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!installTransmissionCurves.py /home/jmyles/DATA\n",
```

Comment:
This cell makes me realize, it would be really nice to have a link to the source of each command line task in the markdown cell above the cell  where you run or unpack that task, for reference.